pages/gnosis/gnosis.md

```diff
@@ -0,0 +1,43 @@
+---
+name: "Gnosis"
+logo: "gnosis.svg"
+color: "#f3f2f2"
+schemas:
+  - "gnosis"
+links:
+  - name: "Website"
+    link: "https://gnosis.io/"
+  - name: "Gnosis Safe"
+    link: "https://gnosis-safe.io/"
+  - name: "Gnosis Protocol"
+    link: "https://docs.gnosis.io/protocol/"
+  - name: "Github"
+    link: "https://github.com/gnosis"
+tags:
+  - "Gnosis Safe"
+  - "Gnosis Protocol"
+  - "DEX"
+  - "DeFi"
+  - "IDO"
+dashboards:
+  - "gnosis-safe_2"
+  - "HtS7kMiZWYS2bxA5oArxlevFwX4B0O3ZZAiRxcd3"
+  - "gnosis-protocol-custom-market-making"
+embeds:
+  - name: "Gnosis Safe Total Value Locked"
+    link: "https://explore.duneanalytics.com/embed/query/5059/visualization/9969?api_key=d4YzPKwiyQGjBfvHMu9QlA23a4ZdlGBTz7Et01nw"
+  - name: "Gnosis Protocol Daily Trading Volume"
+    link: "https://explore.duneanalytics.com/embed/query/8790/visualization/17606?api_key=jN8q3XqC8sWUnHw2uzLciDLXNPmv0I4IDYcJA5W3"
+---
+
+Gnosis builds new market mechanisms for open finance.
+
+Global interconnection has proved that open access to information, resources, and financial infrastructure is key for global resilience. By designing fairer marketplaces, Gnosis aims to build a more evenly distributed future. Our three interoperable product lines allow you to securely create, trade, and hold crypto assets.
+
+**Create:** Gnosis developed an open framework for creating crypto assets that represent information about future events with conditional outcomes. These conditional tokens can be integrated with prediction markets and other applications.
+
+**Trade:** Gnosis Protocol is a fully permissionless DEX, which has been in research and development over the course of the last two years. Gnosis Protocol enables ring trades to maximize liquidity.
+
+**Hold:** The Gnosis Safe Multisig is the most trusted platform to store digital assets on Ethereum and the only formally verified smart contract wallet.
```

Comment:
I know that exactly what's written on gnosis.io, but I would prefer if we just state "The Gnosis Safe Multisig is the most trusted platform to manage digital assets on Ethereum.", i.e. `s/store/manage` and remove the part about formal verification since that was just done for the first version and not afterwards anymore.